git-release: add page (#6095)
tldr-pages/tldr: pages/common/git-release.md

# git release

> Create a Git release tag.
> Part of `git-extras`.
> More information: <https://github.com/tj/git-extras/blob/master/Commands.md#git-release>.

> - Create a Git release, locally and remotely:

Reviewer: What do you think of this? It's also a bit more accurate.

```suggestion
- Create and push a Git release:
```

`git release {{tag_name}}`

> - Create a signed Git release, locally and remotely:

Author: ```suggestion
- Create and push a signed Git release:
```

`git release {{tag_name}} -s`

> - Create a Git release with a message, locally and remotely:

Author: ```suggestion
- Create and push a Git release with a message:
```
